routes/cohortsRoute.js

```diff
@@ -0,0 +1,64 @@
+const express = require('express');
+const route = express.Router();
+
+const db = require('../data/dbConfig.js')
+
+route.get('/', async (req, res) => {
+    try {
+        const cohorts = await db('cohorts'); 
+        res.status(200).json(cohorts)  
+    } catch (error) {
+        res.status(500).json(error)
+    }
+})
+
+route.get('/:id', async (req, res) => {
+    try {
+        const cohort = await db('cohorts').where({id: req.params.id}).first('name');
+        res.status(200).json(cohort)
+    } catch (error) {
+        res.status(500).json(error)
+    }
+})
+
+route.get('/:id/students', async (req, res) => {
+    const id = req.params.id;
+    try {
+        const cohortStudents = await db('cohorts').where({ id :students.cohort_id}).innerJoin('students');
+        res.status(200).json(cohortStudents);
+    } catch (error) {
+        res.status(500).json(error)
```

Comment:
I was only able to get the 500 error here with no error message being returned. Double check the logic in your `try` block